scripts/read_conda_versions.sh

```diff
@@ -0,0 +1,181 @@
+###   dependencies:
+###       libcxx >=4.0.1
+###       libprotobuf >=3.5.1,<3.6.0a0
+###       python >=3.6,<3.7.0a0
+###       six
+
+# Echo what packages we'll look through.
+echo "Processing these packages:"
+for pkg in "${conda_search_packages[@]:2}"; do
+  echo "  $pkg"
+done
+
+
+
+#
+# Look up each package in conda info, then install it and search the exported
+# symbols for signs of cxx11
+#
+for pkg in "${conda_search_packages[@]:2}"; do
+  echo "Processing $pkg" >> $VERBOSE_LOG
+
+  # Split each line into an array and build the package specification
+  # <package_name (1st line only)>  maj.min.patch  build_string  channel_name
+  line_parts=( $pkg )
+  if [[ ${line_parts[0]} == $PKG_NAME ]]; then
+    # First line of output
+    PKG_VERSION="${line_parts[1]}"
+    PKG_BUILD_STR="${line_parts[2]}"
+  else
+    PKG_VERSION="${line_parts[0]}"
+    PKG_BUILD_STR="${line_parts[1]}"
+  fi
+  PKG_SPEC="$PKG_NAME=$PKG_VERSION=$PKG_BUILD_STR"
+
+  # Output current pkg spec
+  echo
+  echo "${PKG_SPEC}:"
+  echo "Determined that the package spec is $PKG_SPEC" >> $VERBOSE_LOG
+
+  # Split the output of conda_info into an array of lines
+  pkg_dependencies=()
+  while read -r line; do pkg_dependencies+=("$line"); done <<< "$(conda info "$PKG_SPEC" $CONDA_CHANNEL)"
+
+  # List all the listed dependencies in `conda info`
+  if [ "${#pkg_dependencies[@]}" -gt 19 ]; then
+    echo "  Listed dependencies:"
+    echo "  Listed dependencies:" >> $VERBOSE_LOG
+    for pkg_dependency in "${pkg_dependencies[@]:20}"; do
+      echo "    $pkg_dependency"
+      echo "    $pkg_dependency" >> $VERBOSE_LOG
+    done
+  else
+    echo "  No listed dependencies in conda-info" >> $VERBOSE_LOG
+  fi
+
+  # But sometimes (a lot of the time) the gcc with which a package was built
+  # against is not listed in dependencies. So we try to figure it out manually
+  # We install this exact package, and then grep the exported symbols for signs
+  # of cxx11
+  echo "Calling conda-uninstall on $PKG_NAME" >> $VERBOSE_LOG
+  echo "conda uninstall -y $PKG_NAME --quiet" >> $VERBOSE_LOG
+  conda uninstall -y "$PKG_NAME" --quiet >> $VERBOSE_LOG 2>&1
+
+  echo "Calling conda-install on $PKG_SPEC" >> $VERBOSE_LOG
+  echo "conda install -y $PKG_SPEC --quiet --no-deps $CONDA_CHANNEL" >> $VERBOSE_LOG
+  conda install -y "$PKG_SPEC" --quiet --no-deps $CONDA_CHANNEL >> $VERBOSE_LOG 2>&1
+  if [ $? -eq 0 ]; then
+    # Only grep the exported symbols if the library was installed correctly
+
+    MENTIONS_CXX11="$(nm "$CONDA_ROOT/lib/$PKG_INSTALLED_LIB" | grep cxx11 | wc -l)"
+    if [ $MENTIONS_CXX11 -gt 0 ]; then
+      echo "  This package is built against the recent gcc ABI ($MENTIONS_CXX11 mentions of cxx11)"
+      echo "$CONDA_ROOT/lib/$PKG_INSTALLED_LIB mentions cxx11 $MENTIONS_CXX11 times" >> $VERBOSE_LOG
+    fi
+  else
+    echo "Error installing $PKG_SPEC , continuing"
+    echo "Error installing $PKG_SPEC , continuing" >> $VERBOSE_LOG
+  fi
+done
```

Comment:
This is the script I used to scrape all the conda packages for mentions of cxx11 to guess which gcc ABI they had. It is not called by anything.